README.md

```diff
@@ -67,3 +74,22 @@ Press CTRL+C to quit
  * Debugger PIN: 113-666-066
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running Tests
+Selenium tests run using a Chrome headless browser. Chrome may need to be installed on your system for these tests to work.
+
+To run the full test suite, execute the `pytest` command from the project root.
```

Comment:
I'm a big fan of suggesting `poetry run pytest` here instead of just `pytest` - the latter is fine as long as you absolutely trust your devs to remember to activate their virtual environment first, and I certainly don't trust that in myself!